src/App.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => (
+    title.toLowerCase().includes(query.toLowerCase())
```

Comment:
```suggestion
    title.toLowerCase().includes(query.toLowerCase().trim())
```
<img width="457" alt="image" src="https://user-images.githubusercontent.com/55802630/184603451-0df4d44f-7ba2-4c47-8acb-ba95387ee68c.png">